zerver/openapi/python_examples.py

```diff
@@ -344,6 +344,25 @@ def create_reusable_invitation_link(client: Client) -> None:
     validate_against_openapi_schema(result, "/invites/multiuse", "post", "200")
 
 
+@openapi_test_function("/invites/{prereg_id}:delete")
```

Comment:
We should probably spell it `invite_id` rather than `prereg_id` in the API documentation; `PreregistrationUser` is an internal to the server concept and not one that makes sense to expose to clients.

Reply:
OK, I have changed `prereg_id` to `invite_id`.